task/pnc-prebuild-git-clone-oci-ta/0.1/patch.yaml

```diff
@@ -0,0 +1,73 @@
+- op: replace
+  path: /metadata/name
+  value: pnc-prebuild-git-clone-oci-ta
+- op: replace
+  path: /metadata/annotations/tekton.dev~1displayName
+  value: pnc prebuild git clone oci trusted artifacts
+- op: replace
+  path: /spec/description
+  value: The pnc-prebuild-git-clone-oci-ta task will clone a repo from the provided url, apply
+    PNC prebuild modifications (from https://github.com/project-ncl/konflux-tooling) and store
+    it as a trusted artifact in the provided OCI repository. The prebuild modifications create
+    a Containerfile and suitable build script in order for the Java based project to be built
+    within a container given build parameters configured from PNC.
+
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_SCRIPT
+    description: Middleware (Maven/Gradle/Ant/SBT) build script to build the project to embed with the Containerfile
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_TOOL
+    description: The build tool to use (ant, gradle, maven, sbt).
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_TOOL_VERSION
+    description: The build tool version to use (e.g. 3.9.5)
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: JAVA_VERSION
+    description: Java version to use (7, 8, 9, 11, 17, 21, 22, 23)
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: RECIPE_IMAGE
+    description: The image from the build recipe to use
+    type: string
+
+- op: add
+  path: /spec/steps/2
+  value:
+    name: preprocessor
+    image: quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling:latest
+    securityContext:
+      runAsUser: 0
+    computeResources:
+      limits:
+        cpu: 300m
+        memory: 512Mi
+      requests:
+        cpu: 10m
+        memory: 512Mi
+    args:
+      - prepare
+      - --build-tool-version=$(params.BUILD_TOOL_VERSION)
+      - --java-version=$(params.JAVA_VERSION)
+      - --recipe-image=$(params.RECIPE_IMAGE)
+      - --tooling-image=quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling:latest
```

Comment:
I don't think that we should reference an image from redhat-user-workloads as this can be accidentally deleted of the component is deleted. Can we release this to be an image in `quay.io/konflux-ci` instead?

Reply:
Agreed but we'll have to discuss that on how best to release that single component to the konflux owned namespace.